App/Services/EventHandlers/TextileNodeEventHandler.ts

```diff
@@ -1,8 +1,8 @@
 import { Store } from 'redux'
 
 import { ILocalPhotoResult } from '../../Models/TextileTypes'
-import {  Update, ThreadUpdate, BlockType, NotificationInfo } from '../../NativeModules/Textile'
-import EventEmitter from '../../NativeModules/Events'
+import {  Update, ThreadUpdate, BlockType, NotificationInfo } from '@textile/react-native-sdk'
+import { Events } from '@textile/react-native-sdk'
```

Comment:
Multiple import statements from @textile/react-native-sdk here, combine them. Unless there's some reason you want the separate imports?

Reply:
nah, just did a lot of find/replaces so missed those